src/Core/src/Core/ICanReturnVisualTreeElement.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Maui
+{
+	interface ICanReturnVisualTreeElement
```

Comment:
Do we still use this "IVerb" naming anywhere else?

Reply:
IVisualTreeElementProvidable